### ISSUE TITLE
disable theme builder when running locally

### ISSUE THREAD

**Issue by timveil** [gitpitch/gitpitch#296]
the theme builder icon seems to always be present when running a presentation locally using the desktop manager.  I'd like the ability to disable this feature.  Most collaborators on our presentations are creating content not modifying theme or style.  I'd like to prevent them from accidentally fiddling with theme builder and committing unwanted changes to PITCHME.yaml

**Comment by gitpitch** [comment]
I like this feature so I've gone ahead and implemented a `theme-builder` setting that can now be used in the `PITCHME.yaml` for any slide deck. When this setting is disabled the theme builder icon is hidden, for example:

```
theme-builder : false
```

Note, while the icon is hidden the theme builder is still available by pressing the `T` key on your keyboard. So functionally it's still available but it is no longer front and center. This seems to make sense. Would you agree? The alternative would to be disable all access to the theme builder if this setting is disabled.

I still need to add these details to the docs by you can go ahead and try out the feature now. First you'll need to download the [latest version of the Desktop](https://gitpitch.com/docs/work-offline/desktop-faq).

**Comment by timveil** [comment]
@gitpitch i think your change makes sense.  i can see how completely disabling would be useful too but this is good first step.

**Comment by gitpitch** [comment]
Having given this some further thought I have decided to make the behavior on the new `theme-builder` explicit. **When the setting is disabled the theme builder tool is disabled for the deck.**

```yaml
theme-builder : false
```

This removes any ambiguity. And prevents unintended discovery and use of the tool. Which is the original purpose of the setting. The desktop has been updated accordingly. You'll need to pull the desktop latest image to test. Feedback welcome. Thanks, David.
